framework/assets/yii.gridView.js

```diff
@@ -123,7 +123,7 @@
             var $form = $('<form/>', {
                 action: url,
                 method: 'get',
-                class: 'gridview-filter-form',
+                'class': 'gridview-filter-form',
```

Comment:
https://www.dropbox.com/s/6uxukanl7jy4sn0/%D0%A1%D0%BA%D1%80%D0%B8%D0%BD%D1%88%D0%BE%D1%82%202016-05-18%2014.43.58.png?dl=0